components/nextdoor/actions/create-ad-group/create-ad-group.mjs

```diff
@@ -0,0 +1,129 @@
+import app from "../../nextdoor.app.mjs";
+import utils from "../../common/utils.mjs";
+
+export default {
+  key: "nextdoor-create-ad-group",
+  name: "Create Ad Group",
+  description: "Creates an ad group based on the input payload for an existing campaign. [See the documentation](https://developer.nextdoor.com/reference/adgroup-create).",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    app,
+    advertiserId: {
+      propDefinition: [
+        app,
+        "advertiserId",
+      ],
+    },
+    campaignId: {
+      propDefinition: [
+        app,
+        "campaignId",
+        ({ advertiserId }) => ({
+          advertiserId,
+        }),
+      ],
+    },
+    name: {
+      description: "The name of the ad group.",
+      propDefinition: [
+        app,
+        "name",
+      ],
+    },
+    placements: {
+      type: "string[]",
+      label: "Placements",
+      description: "The placements for the ad group.",
+      options: [
+        "RHR",
+        "FEED",
+        "FSF",
+      ],
+    },
+    bid: {
+      type: "object",
+      label: "Bid",
+      description: "The bid for the ad group.",
+      default: {
+        amount: "USD 10",
+        pricing_type: "CPM",
+      },
+    },
+    budget: {
+      type: "object",
+      label: "Budget",
+      description: "The budget for the ad group. Both `amount` and `budget_type` properties are required.",
+      default: {
```

Comment:
Is it possible to split this into two props (Budget Amount and Budget Type) or can the object have more properties?